### DISCUSSION TITLE
"Component hash" in SPDX

### DISCUSSION THREAD

**Discussion by dgutson** [oss-review-toolkit/ort#7064] [Q&A] [answered]
Hi,

  how can we enable the "component hash" in the SPDX report? I can't find it in the documentation.

**Comment by sschuberth** [comment, marked as answer]
The NTIA "Component Hash" [maps to](https://spdx.github.io/spdx-spec/v2.3/how-to-use/#k22-mapping-ntia-minimum-elements-to-spdx-fields) the SPDX "[Package Checksum](https://spdx.github.io/spdx-spec/v2.3/package-information/#710-package-checksum-field)" which we're not writing out yet (we're only writing out the SPDX "[Package Verification Code](https://spdx.github.io/spdx-spec/v2.3/package-information/#79-package-verification-code-field)", but that's a different thing). I'll prepare a PR.

**Reply by sschuberth** [reply]
See https://github.com/oss-review-toolkit/ort/pull/7066.

**Reply by dgutson** [reply]
you rock

**Comment by dgutson** [comment]
@sschuberth because of https://github.com/oss-review-toolkit/ort/pull/7066 is this enabled now? Should I just update ORT and checksums will start to show up?

**Reply by sschuberth** [reply]
Yes.

**Comment by dgutson** [comment]
@sschuberth any chance this could be also added to the HTML web page too?

**Reply by sschuberth** [reply]
You mean the web-app report? I'm not a JavaScript guy, so someone else would have to do that. PRs welcome.

**Reply by dgutson** [reply]
OK, I'll have interns hopefully starting in a few weeks.
